api/src/main/java/org/openmrs/api/ConceptService.java

```diff
@@ -393,6 +393,8 @@
 	 * 
 	 * @param includeRetired If <code>true</code> then the search will include voided Drugs
 	 * @return A List&lt;Drug&gt; object containing all matching Drugs
+	 * @should return a list of drugs when include retired is true
+	 * @should return a list of drugs when include retired is false
```

Comment:
@mavrk you didnt write this as suggested.
you should be more specific, what is the expected behavior of the method given the different parameter.

* should return all drugs including retired ones if given true
* should return all drugs excluding retired ones if given false